### ISSUE TITLE
Sent messages do not show up (nowhere) in email account

### ISSUE THREAD

**Issue by Neudrino** [deltachat/deltachat-core#15]
I experience a problem, that outgoing / sent messages are nowhere to be seen / saved in the email account. I think this is not desired?
It would be great, if sent messages could be copied to "sent" folder, as it is usually done for all normally sent emails (for a start).

However, as soon as one uses DeltaChat extensively this might lead to an "message overflow" in the inbox. So maybe one could also define a folder, where chats are stored, such that outgoing messages are moved there, and incoming messages too. This would avoid cluttering the normal inbox, with chat snippets.

Thanks for the nice app!

**Comment by r10s** [comment]
Sent messages should go to the "Chats" folder which is created by Delta Chat to avoid "message overflow" on extensively use.

Incoming messages from other messengers are also copied to this folder as soon as they're read by Delta Chat.

**Comment by Neudrino** [comment]
Then it has probably to do with the email provider I was using. The provider (freenet) does not allow to create top level directories, but all the dirs are subdirs of INBOX, such that the app might have been unable to create a dir, I guess.

Maybe one could allow to define a custom prefix somewhere in config?

**Comment by kskarthik** [comment]
I have no problem with account on disroot.org I receive all deltachat mails in *chat* folder 

Gmail has this problem, however

**Comment by JRHaigh** [comment]
Fine on Gmail, but remember that on many email clients you have to manually refresh your folder list and subscribe to the new Chats folder.
&nbsp;&nbsp;&nbsp;&nbsp;I had the same problem when I was trying to get Delta Chat to work a couple of months ago. I didn't know what to look for, but I couldn't see any new folder appear in my IMAP tree. However, because an SMTP port issue was preventing messages from sending (without any error messages at the time), I don't think Delta Chat had created the folder yet. When I refreshed my IMAP folder list after fixing the port issue, there was the Chats folder at the root of the IMAP tree.

**Comment by r10s** [comment]
Is this problem still present?

If it's Gmail, please refer to https://github.com/deltachat/deltachat-android/issues/20 and close this issue.

**Comment by Neudrino** [comment]
Yes, it is.
I do _not_ use google. It has nothing to do with IMAP sync. I checked also with web interface.

With regard to @kskarthik :
Maybe it was my fault to mention 2 problems in one report. However, this was originally about outgoing messages not saved.

I did testing with **provider _freenet.de_** again:
A "Chats" subfolder is not created.
Outgoing messages are saved now, but in an arbitrary preexisting folder (I guess the first in folder list).
Incoming messages remain in the inbox.

I did testing with **provider _gmx.de_** again:
What worked previously on this provider is now broken. In Detail:
"Chats" subfolder is created. However, when sending a message
_Error: Cannot append message to "Chats", error #11_
shows. Sent messages are not saved (nowhere).

**Summary:**
Maybe it would be more precise to rename the report to
_"Chats" subfolder not created correctly_
or such? It seems to me, that not having the proper folders is causing trouble?
